crud/options.go

```diff
@@ -63,6 +63,25 @@ func (opt OptInt) Get() (int, bool) {
 	return opt.value, opt.exist
 }
 
+// OptFloat64 is an optional float64.
+type OptFloat64 struct {
```

Comment:
Does all float64 numbers could be used as a Lua number type (as example math.MaxFloat64)?
We need a note here if not.

Reply:
https://www.lua.org/pil/2.3.html
> The number type represents real (double-precision floating-point) numbers.

Lua uses 64-bit floats for a number, so everything should be fine.